modules/mediawiki/files/cookbooks/rename-wiki.py

```diff
@@ -0,0 +1,53 @@
+import subprocess
+import argparse
+
+# Define the mapping of db clusters to db names
+db_clusters = {
+    'c1': 'db131',
+    'c2': 'db101',
+    'c3': 'db142',
+    'c4': 'db121',
+    'c5': 'db131',
+}
+
+
+def get_db_cluster(oldwiki_db):
+    command = f"salt-ssh -E 'db131*' cmd.run 'USE mhglobal; SELECT wiki_dbcluster FROM cw_wikis WHERE wiki_dbname = \"{oldwiki_db}\"'"
```

Comment:
Has this command been tested?

Reply:
Not with salt but the actual command is the one I always use. I'll test them each individually of course.

Reply:
Check the full thing you've typed out. Is missing mysql -e